### PR TITLE
New version: PrismLauncher.PrismLauncher version 6.3

### PR DESCRIPTION
**Pull request by PrismAutomata**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95555)

### CODE REVIEW

**Review by russellbanks — APPROVED** [review]

**Review by russellbanks — DISMISSED** [review]

File: manifests/p/PrismLauncher/PrismLauncher/6.3/PrismLauncher.PrismLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+PackageVersion: "6.3"
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-02-03
+Installers:
```
Comment (stephengillie) [inline manifests/p/PrismLauncher/PrismLauncher/6.3/PrismLauncher.PrismLauncher.installer.yaml:14]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/p/PrismLauncher/PrismLauncher/6.3/PrismLauncher.PrismLauncher.installer.yaml:14]: The package installs, but requires the dependency to run:
image

**Comment by Scrumplex** [comment]
> This package appears to require dependencies in order to install successfully.
> 
> This PR is blocked until support for dependencies is implemented in:

Any reason for this to be blocked? There is a security fix in this release, so it would be advantageous to get this released. This dependency has been there for the previous releases as well, it was just not declared before.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Hi @Scrumplex,

> Any reason for this to be blocked? There is a security fix in this release, so it would be advantageous to get this released. This dependency has been there for the previous releases as well, it was just not declared before.

Very sorry for the delay. I made an error in setting this aside with new packages with dependencies, instead of manually validating it.

**Pull request merged**